ReportingServicesTools.psd1

```diff
@@ -108,34 +119,34 @@
 	FileList = ''
 	
 	PrivateData = @{
-    # PSData is module packaging and gallery metadata embedded in PrivateData
-    # It's for rebuilding PowerShellGet (and PoshCode) NuGet-style packages
-    # We had to do this because it's the only place we're allowed to extend the manifest
-    # https://connect.microsoft.com/PowerShell/feedback/details/421837
-    PSData = @{
-        # The primary categorization of this module (from the TechNet Gallery tech tree).
-        Category = "Databases"
+        # PSData is module packaging and gallery metadata embedded in PrivateData
+        # It's for rebuilding PowerShellGet (and PoshCode) NuGet-style packages
+        # We had to do this because it's the only place we're allowed to extend the manifest
+        # https://connect.microsoft.com/PowerShell/feedback/details/421837
```

Comment:
Actually, it was in your original manifest already, didn't check for it then. Looked it up right now and neither on connect nor uservoice can I find a matching issue.
In my upcoming update of the PR I updated this link to point at the manifest article on msdn:
https://msdn.microsoft.com/en-us/library/dd878337(v=vs.85).aspx